packages/react-reconciler/src/ReactFiberWorkLoop.old.js

```diff
@@ -1912,6 +1913,15 @@ function commitRootImpl(root, renderPriorityLevel) {
     firstEffect = finishedWork.firstEffect;
   }
 
+  if (root.clearContainerBeforeMount) {
+    // We are about to mount into a container that previous contained non-React elements.
+    // We should clear the previous contents before beginning.
+    // This mimics legacy render into subtree behavior in a way that is safe for concurrent mode.
+    // (It doesn't result in multiple obsevable mutations.)
+    root.clearContainerBeforeMount = false;
+    clearContainer(root.containerInfo);
+  }
```

Comment:
Feels a little goofy to check for this here. Any better suggestions?

Reply:
I was going to say maybe we don't need to make it an option. Could just be the default like how ReactDOM.render works.

Reply:
I was confused.